### DISCUSSION TITLE
Any plans to add a preferences window?

### DISCUSSION THREAD

**Discussion by Mr0Bread** [zed-industries/zed#6501] [Feature Requests]
JSON is cool, but having GUI window, ideally, searchable like in JetBrains IDEs, would be really cool

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
